test/mitmproxy/utils/test_data.py

```diff
@@ -1,7 +1,8 @@
+import pytest
 from mitmproxy.utils import data
-from . import tutils
 
 
 def test_pkg_data():
     assert data.pkg_data.path("tools/console")
-    tutils.raises("does not exist", data.pkg_data.path, "nonexistent")
```

Comment:
I'm -0.001 on this. The string-based exception stuff can be nice, so moving to pytest.raises would be a slight net negative. If we can make an argument that the context handler pytest.raises offers would be useful, I might be +0.001.